core/src/main/java/org/apache/carbondata/core/reader/CarbonIndexFileReader.java

```diff
@@ -112,4 +112,34 @@ public void openThriftReader(byte[] fileData) throws IOException {
   public boolean hasNext() throws IOException {
     return thriftReader.hasNext();
   }
+
+  /**
+   * Return the footer offset of the given dataFileName in the index file
+   *
+   * @param indexFilePath path of the index file
+   * @param dataFileName file name of the data file to match in the index file
+   * @param conf hadoop configuration
+   * @return footer offset of the data file
+   * @throws IOException
+   */
+  public static long readFooterOffsetFromIndexFile(String indexFilePath, String dataFileName,
```

Comment:
This method is unused..Please remove if not needed

Reply:
fixed